vardict.pl

```diff
@@ -4086,7 +4107,13 @@ sub findconseq {
 	my $maxq = 0;
 	my $mnt = "";
 	my $tt = 0;
-	while(my ($nt, $ncnt) = each %$nv) {
+	my @tmp = ();
```

Comment:
The sort by bases with the same counts was added. Another approach can be to check all the bases if their counts are the same and create candidate consensus for each of the sequences, then check if the reference contains such sequence (and skip such sequences for softclips to avoid REF=ALT issue).